woocommerce/woocommerce-android
# Fix dark mode issues

Part of: #12281

### Description
This PR fixes two small issues in dark mode. 

The first one is the purchase button elevation. In dark mode, the bottom section elevation differs from the bottom sheet, making the UI look weird. In this PR, I fix that by using different elevations depending on whether the device is in dark mode or expanded. 

The second issue was the selected rate background in dark mode.

### Testing information

1. Create an Order with the processing status on a store that can create shipping labels
2. Tap on Orders
3. Tap on the order created on step 1
4. Tap on create shipping labels
5. Check that the shipment details section works as expected
6. Tap on a shipping rate
7. Check that the selected background is the expected
8. Turn on dark mode
9. Check the UI is readable and that the shipment section and the purchase button have the same elevation

### The tests that have been performed
- Testing the UI works as expected

### Images/gif
| Before  | After |
| ------------- | ------------- |
| <img width="300" src="https://github.com/user-attachments/assets/0d18fc94-4081-4529-9ce6-2584eed3f5ad" />  | <img width="300" src="https://github.com/user-attachments/assets/eb9340da-d838-4994-a2d7-c6e86aea02f3" />  |
| <img width="300" src="https://github.com/user-attachments/assets/b5af5648-7548-48ab-a98e-94ad5bf97397" />  | <img width="300" src="https://github.com/user-attachments/assets/0edd61f3-3507-4749-a5b2-e0ded681979f" />  |



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
